conftest.py

```diff
@@ -210,13 +211,16 @@ def database_table_fixture(request):
     filepath = params.get("filepath", "")
 
     database_name = params["database"]
-    database_name_to_conn_id = {Database.SQLITE: "sqlite_default"}
+    database_name_to_conn_id = {
+        Database.SQLITE: "sqlite_default",
+        Database.BIGQUERY: "google_cloud_default",
+    }
     conn_id = database_name_to_conn_id[database_name]
     database = create_database(conn_id)
 
     database.drop_table(table)
     if filepath:
-        database.load_file_to_table(filepath, table)
+        database.load_file_to_table(File(str(filepath)), table)
```

Comment:
@utkarsharma2, what if we changed the fixture to receive `File`? This way, the user can customize more parameters if they want to..!
Also, did you find any limitations with having a single fixture for both the database and table? This was something I was unsure when I introduced the fixture - if we have a need, we can always break it into two..!

Reply:
1. Sounds good, I'll make the change. 
2. We were actually always using a table in the context of a certain DB and I think that's why I haven't seen any limitations as of now.